ts/parsers/pptGlobalsParser.ts

```diff
@@ -11,4 +12,27 @@ export default class PptGlobalsParser {
             throw error;
         }
     }
+
+    public static async getSections(pptFilePath: string): Promise<PptSection[]> {
+        try {
+            const slideShowGlobals = await FileHandler.parseContentFromFile(join(pptFilePath, "ppt/presentation.xml"));
+
+            const allSections = getAttributeByPath(slideShowGlobals, ["p:presentation", "p:extLst", "p:ext", "p14:sectionLst", "p14:section"], []);
+
+            let slidesCount = 0;
+            return allSections.map(section => {
+                const sectionSlidesLength = section["p14:sldIdLst"][0]["p14:sldId"].length;
+                const startingSlide = slidesCount;
+                slidesCount += sectionSlidesLength;
+
+                return {
+                    title: section["$"]["name"],
+                    startingSlide: startingSlide,
+                    lastSlide: slidesCount -1
```

Comment:
Edge case, what if there are no slides in the section?

Reply:
Good catch @MubashirAR, thanks
Fixed with this commit
https://github.com/rabi92/airppt-parser/pull/15/commits/3e736a8c0f91796bd1ef6d3e891292d15bf59e3b